README.md

```diff
@@ -31,6 +31,236 @@ Application-level tracing for Rust.
 structured, event-based diagnostic information. `tracing` is maintained by the
 Tokio project, but does _not_ require the `tokio` runtime to be used.
 
+## Usage
+
+(The below example is borrowed from the `log` crate's yak-shaving
+[example](https://docs.rs/log/0.4.10/log/index.html#examples), modified to
+idiomatic `tracing`)
+
+In `Cargo.toml`:
+
+```toml
+[dependencies]
+tracing = "0.1"
+tracing-subscriber = "0.2.0-alpha.2"
+```
+
+In `src/main.rs`:
+
+```rust
+mod yak_shave;
+
+use std::{error::Error, io};
+use tracing::{debug, error, info, span, warn, Level};
+use tracing_subscriber::FmtSubscriber;
+use yak_shave;
+
+fn main() {
+    // a builder for `FmtSubscriber`.
+    let subscriber = FmtSubscriber::builder()
+        // all spans/events with a level higher than DEBUG (e.g, info, warn, etc.)
+        // will be written to stdout.
+        .with_max_level(Level::DEBUG)
+        // completes the builder.
+        .finish();
+
+    tracing::subscriber::set_global_default(subscriber)
+        .expect("setting defualt subscriber failed");
+
+    let number_of_yaks = 3;
+    // this creates a new event, outside of any spans.
+    info!(number_of_yaks, "preparing to shave yaks");
+
+    let number_shaved = yak_shave::shave_all(number_of_yaks);
+    info!(
+        all_yaks_shaved = number_shaved == number_of_yaks,
+        "yak shaving completed."
+    );
+}
+```
+
+In `src/yak_shave.rs`:
```

Comment:
TIOLI: I don't know if it's really necessary to show multiple files in the example...I split the `examples/` dir version just so that some of the code could be used in multiple examples...

Reply:
yeah, removed that stuff.